site/src/pages/index.module.css

```diff
@@ -0,0 +1,64 @@
+/**
+ * CSS files with the .module.css suffix will be treated as CSS modules
+ * and scoped locally.
+ */
+
+.banner {
+  padding-top: 5rem;
+  padding-bottom: 1.8rem;
+  text-align: center;
+  position: relative;
+  overflow: hidden;
+  background-color: transparent;
+}
+
+.titleContainer {
+  margin-bottom: 1.5rem;
+}
+
+.title {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  gap: 0.5rem;
+}
+
+.genAITitle {
+  font-size: 4rem;
```

Comment:
It would be nice to have some predefined set of font scales like body, subtitle, title, display, etc and reuse them.
Not sure if it's doable everywhere, as I understand we overriding default styles and we have pure `css`

Reply:
I agree.
Generally I am trying to use default theme styles to have minimum changes in css, but there are some cases when they should be changed (like here for logo)